src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -39,6 +39,7 @@ function LineNumberTooltip({
     targetNode: HTMLElement;
     lineNumber: number;
   }) => {
+    trackEvent("editor.gutter_mouse_over");
```

Comment:
Our last tracker only tracked an event once the tooltip loaded (read: `n hits` displayed). This is more general purpose.